README.rst

```diff
@@ -91,6 +89,11 @@ NI-DCPower Python API Status
     :target: http://pypi.python.org/pypi/nidcpower
 
 
+.. |nidcpowerDocs| image:: https://readthedocs.org/projects/nidcpower/badge/?version=latest
+    :alt: NI-DCPower Documentation Status
```

Comment:
consider 

NI-DCPower Python Bindings Documentation Status

or

NI-DCPower Python API Documentation Status

or

`nidcpower` Python module Documentation Status.

In short, make it so it is specific to Python. Use terminology that is consistent with whatever we use elsewhere.

Reply:
The section headings here say "... Python API Status" so Python API Documentation Status seems appropriate.